addons/arsenal/functions/fnc_addDeafultLoadout.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: 654wak654
+ * Adds a loadout to the "Default Loadouts" list.
+ *
+ * Arguments:
+ * 0: Name of loadout <STRING>
+ * 1: getUnitLoadout array <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["Squad Leader", getUnitLoadout sql1] call ace_arsenal_fnc_addDefaultLoadout
+ *
+ * Public: Yes
+*/
+
+params [["_name", "", [""]], ["_loadout", [], [[]], 10]];
+
+GVAR(defaultLoadoutsList) append [_name, _loadout];
```

Comment:
Maybe some check to not add same name twice? Not sure if that matters.

Reply:
true, completely forgot about that too, you definitely need name checking in there.

Reply:
I can't identify loadouts by magic, I had to use something (plus it makes sense)